src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/existing_class_def_1.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definition class="Symfony\Bundle\FrameworkBundle\Tests\Console\Descriptor\ClassWithDocComment" public="false" synthetic="false" lazy="false" shared="true" abstract="false" autowired="false" autoconfigured="false" file="">
+  <description><![CDATA[This is a class with a doc comment.]]></description>
+</definition>
+
```

Comment:
now it's a line to many :P not sure it makes tests fail

Reply:
No, there is trim and trim like operations which prevent that.

Removed last blank line in xml, it seems the blank lines that Phpstorm shows me don't match to the reality :/